code/modules/heavy_vehicle/mech_interaction.dm

```diff
@@ -491,3 +491,10 @@
 					return TRUE
 			L.apply_damage(legs.trample_damage, BRUTE)
 			return TRUE
+
+/mob/living/heavy_vehicle/proc/SetLockdown(var/state = 1)
+	lockdown = state
```

Comment:
You can probably make this proc simpler by making it lockdown = !lockdown and taking the state stuff out. That way it's a toggle instead of an argument based proc.

Reply:
Sure, I suppose it doesn't need to be more complicated than that. Changed to a toggle.